client/elements/menus/sc-language-menu.html

```diff
@@ -184,6 +189,18 @@
               this.loadResources(this.resolveUrl(`/localization/elements/sc-language-menu/${this.language}.json`));
           }
 
+          _computeLanguageMenu() {
+            const languageMenu = [];
+            for (let item in this.languageChoice) {
+              if (this.languageChoice[item].lang in this.rootLangMappings) {
+                languageMenu.splice(0, 0, this.languageChoice[item]);
```

Comment:
hmm, shouldn't `this.languageChoice[item]` be just `item`?

Reply:
item is just a number. So why would you want to add a number to the first place in the array rather than the whole object?

Reply:
Oh, you're right. I got too used to `forEach` loops. I think we should use one here.

Reply:
Also, if it's an index, it should be named `itemIndex` and not `item`

Reply:
OK, will use `forEach`. Looks indeed a bit better.